lib/rdma.pm

```diff
@@ -0,0 +1,73 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: RDMA(Remote Direct Memory Access) related functions, mainly emulate RDMA
+#
+# To enable RDMA you need a RDMA card or only use emulate function in this lib
+# There are two different emulate technology: rdma_rxe(RDMA over Ethernet emulation), rdma_siw(Software iWARP)
+# Here we use rdma_rxe by default
+#
+# Maintainer: Yong Sun <yosun@suse.com>
+package rdma;
+
+use base Exporter;
+use Exporter;
+use 5.018;
+use strict;
+use warnings;
+use utils;
+use testapi;
+use base 'opensusebasetest';
+use File::Basename;
+use transactional;
+use package_utils;
+
+our @EXPORT = qw(
+  install_rdma_dependency
+  modprobe_rdma
+  link_add_rdma
+  link_add_rxe
+  rdma_record_info
+  enable_rdma_in_nfs);
+
+sub install_rdma_dependency {
+    my @deps = qw(
+      rdma-core-devel
+      librdmacm-utils
+      infiniband-diags
+      libibverbs-utils
+    );
+    my $packages = join(' ', @deps);
+    script_run('zypper --gpg-auto-import-keys ref');
+    install_package($packages);
```

Comment:
I think trup reboot option should be set  https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/214a708458d64b81fe25fad1ca80b69b8c339df0/lib/ssh_crypto_policy.pm#L41

Reply:
Thx, added it